### ISSUE TITLE
[bug] Self relationship drawing line is not aligned

### ISSUE THREAD

**Issue by Dragonol** [TablePlus/diagram-plugin#28]
![image](https://user-images.githubusercontent.com/30685603/112600255-87f70d00-8e43-11eb-9ec1-3ec5d2466c64.png)

**Comment by thien-do** [comment]
Yeah this is hard

**Comment by mittalyashu** [comment]
@thien-do 

![CleanShot 2021-03-29 at 10 27 15](https://user-images.githubusercontent.com/29014463/112788893-82830800-9079-11eb-94fa-18183cc5dde4.gif)

**Comment by thien-do** [comment]
Yeah thank you @mittalyashu . That _may_ be another issue. I'll try to look into both of them in the weekend
